bool msl_in
float64 lat
float64 lon
float64 alt
---
float64 alt_convd